src/collections/workshops/workshops.json

```diff
@@ -0,0 +1,4 @@
+{
+	"permalink": "/initiatives/{{ title | slug }}/",
```

Comment:
You taught me the other day that even though the permalink is defined as using `title` in here, it will be overridden later with the even id. I forgot where the overriding is.

I wonder if it would be more straightforward to directly use `id` rather than `title` here without relying on the overriding?

Reply:
The permalink can be overridden in an individual workshop file (see the value defined for `permalink` [here](https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/d462045ce51890ec82301f7a39af8503ed4b8230/src/collections/workshops/recQcaxqWsxtnljqe.md)) or via the CMS `permalink` field as per https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/468#discussion_r605882949.

I think it's a good idea to default to titles going forward (it makes for friendlier, more readable URLs), but since these workshops have been online for a while and probably linked to in various newsletters etc, I am manually overriding the permalink with the event ID so that the links do not break.

Reply:
So the new workshop records will use `title` in permalinks. I agree using titles is more readable and friendlier.

Reply:
That's right— and if we want we can also change the existing URLs and add redirects. But I thought it would be good to keep this as straightforward as possible without changing URLs as part of the new CMS setup.